src/monitor/pgautofailover.sql

```diff
@@ -37,7 +37,12 @@ CREATE TYPE pgautofailover.replication_state
     'join_primary',
     'apply_settings',
     'prepare_maintenance',
-    'wait_maintenance'
+    'wait_maintenance',
```

Comment:
would be nice to have explanations for these states on `fsm.rst`, and overall we probably need to do one iteration for the documentation of these changes.

I guess deferring that piece to the end of the PR (or even a follow-up PR) makes more sense in case we change any behavior. Just wanted to note here so that we don't forget :)

Reply:
Indeed, that was the plan ;-)